README.md

```diff
@@ -31,15 +32,10 @@ Call `katex.render` with a TeX expression and a DOM element to render into:
 katex.render("c = \\pm\\sqrt{a^2 + b^2}", element);
 ```
 
-To avoid escaping the backslash (double backslash), you can use
-[`String.raw`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/raw)
-(but beware that `${`, `\u` and `\x` may still need escaping):
-```js
-katex.render(String.raw`c = \pm\sqrt{a^2 + b^2}`, element);
-```
-
 If KaTeX can't parse the expression, it throws a `katex.ParseError` error.
 
+To render all math expressions on the page, see [Auto-render Extension](https://khan.github.io/KaTeX/docs/autorender.html).
```

Comment:
This is definitely good to add!  I wonder about no longer having the quick start that most people would use (1 link tag, 2 script tags, then call renderMathInElement). I guess that's what's in https://khan.github.io/KaTeX/docs/autorender.html, but it still might be nice to have the minimal version here...

Reply:
@edemaine I'm not sure about the other two comments.

Reply:
@edemaine I've added a section about the extension.